vowpalwabbit/parse_args.cc

```diff
@@ -435,7 +435,7 @@ input_options parse_source(vw& all, options_i& options)
       .add(make_option("no_daemon", all.no_daemon)
                .help("Force a loaded daemon or active learning model to accept local input instead of starting in "
                      "daemon mode"))
-      .add(make_option("chain_hash", parsed_options.chain_hash)
+      .add(make_option("chain_hash", parsed_options.chain_hash_json)
                .help("enable chain hash for feature name and string feature value. e.g. {'A': {'B': 'C'}} is hashed as "
```

Comment:
should the help message mention this is default and only option for json?

Reply:
Good catch thankyou